common/schema.h

```diff
@@ -370,6 +372,9 @@ namespace swss {
 
 #define CFG_DHCP_TABLE                              "DHCP_RELAY"
 
+#define CFG_FLOW_COUNTER_ROUTE_PATTERN_TABLE        "FLOW_COUNTER_ROUTE_PATTERN_TABLE"
+#define ROUTE_PATTERN_MAX_MATCH_COUNT_FIELD         "max_match_count"
```

Comment:
Should we consider moving the filed definition to routepatternorch.h?
This is not a strong opinion but currently there are conflict styles:
- for most sub orchagents, fields are defined in their own headers and in lower case.
- however, I do see there are some fields definition in schema.h for flexcounter tables and in upper case.
Just FYI.